app/src/main/java/org/oppia/app/profile/AddProfileViewModel.kt

```diff
@@ -13,6 +13,11 @@ class AddProfileViewModel @Inject constructor() : ObservableViewModel() {
   val confirmPinErrorMsg = ObservableField("")
   val nameErrorMsg = ObservableField("")
   val createPin = ObservableField(false)
+  var savedName = ""
+  var savedPin = ""
```

Comment:
All these values are actually not getting used in xml file. Suggest using the concept of data bining to set values via viewmodel. Similar to  `nameErrorMsg`

Reply:
Ok I'll check that